manifests/crds/rollout-crd.yaml

```diff
@@ -15,13 +15,11 @@ spec:
     name: Current
     type: integer
   - JSONPath: .status.updatedReplicas
-    description: Total number of non-terminated pods targeted by this rollout that
-      have the desired template spec
+    description: Total number of non-terminated pods targeted by this rollout that have the desired template spec
```

Comment:
Seemingly `make manifests` removes the added line. Let me know if you'd like me to revert this nonetheless